doc/network.tex

```diff
@@ -1636,7 +1658,9 @@ \subsection{Flow-control and Buffering in the Demultiplexer}
 never overflows.
 If the demultiplexer detects an overflow of the buffer, it means that the peer violated the
 protocol and the MUX/DEMUX layer shuts down the connection to the peer.
```

Comment:
This termination behavior is also described in the section "Pipelining of Mini Protocols", make one reference the other.